highlight.io/components/QuickstartContent/frontend/electron.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+	configureSourcemapsCI,
+	identifySnippet,
+	initializeSnippet,
+	packageInstallSnippet,
+	setupBackendSnippet,
+	verifySnippet,
+} from './shared-snippets'
+
+import { siteUrl } from '../../../utils/urls'
+import { QuickStartContent } from '../QuickstartContent'
+
+const electronInitCodeSnippet = `// hooks.client.ts
+...
+
+import { H } from 'highlight.run';
+
+H.init('<YOUR_PROJECT_ID>', {
+    environment: 'production',
+    version: 'commit:abcdefg12345',
+    tracingOrigins: true,
+	networkRecording: {
```

Comment:
Nit, but looks like there's a mix of tabs and spaces here which is throwing off the indentation for my settings.

<img width="151" alt="Screenshot 2024-07-12 at 7 25 36 AM" src="https://github.com/user-attachments/assets/688a10f8-0894-4c38-83ae-92c0b57e4b7e">